lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -294,6 +294,9 @@ DEBUG_INFO_FLAG ?= -g
 CFLAGS ?= $(DEBUG_INFO_FLAG) -O0 -fno-builtin
 SWIFTFLAGS ?= $(DEBUG_INFO_FLAG) -Onone -Xfrontend -serialize-debugging-options
 
+# Use the just-built standard library
+SWIFTFLAGS += -I $(dirname $(SWIFTC))/../lib/swift/
```

Comment:
Shouldn't the Swift compiler be smart enough to prefer the stdlib from its own resource directory? Does the Swift compiler testsuite also manually specify this by expanding `%swift` like that? And is there a more targeted flag than `-I` to point to the resource directory?

Reply:
You're right, this is not needed. I got into this situation with a Catalyst test, but it turns out I hadn't built the Catalyst modules in my build, so it was falling back to the SDK modules.